### ISSUE TITLE
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### ISSUE THREAD

**Issue by annelfitz** [Esri/mountain-lion-habitat-analysis-js#13]
**Site:** https://esri.github.io/mountain-lion-habitat-analysis-js/
**Date scanned:** 09/15/2026

### Rule
**Rule ID:** color-contrast
**Rule Name:** Elements must meet minimum color contrast ratio thresholds
**Rule Description:** Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
**Standard(s):** WCAG 2.0(AA)
**WCAG Success Criteria:** 1.4.3 Contrast (Minimum) (AA) , EN-301-549 , EN-9.1.4.3
**Impact:** serious
https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=axeAPI

### Issue 1
- **Element location:** div[slot="content-top"] > .map-title-super
- **Element source:** <div class="map-title-super">ArcGIS Maps SDK for JavaScript</div>
- **Suggested fix:** Fix any of the following:
  Element has insufficient color contrast of 2.4 (foreground color: #c56d47, background color: #e3ccb5, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

![Screenshot](https://github.com/Esri/mountain-lion-habitat-analysis-js/raw/allyhawk-assets/.allyhawk/screenshots/e36ad32710627d168eb9c52ab0d86c91.jpg)

**Comment by annelfitz** [comment]
@macandcheese do you think you could take a look at this one? I just ran the Allyhawk tool against this app to try to catch some accessibility violations.
